linode/manager
# feat: [M3-7033] - Update Firewall Landing Device Column

## Description 📝
Updated Firewall Landing Device Column

## Major Changes 🔄
- Changes name of column from 'Linodes' to 'Devices'
- Populated the 'Devices' column with devices instead of linodes
-  Increased the number of visible devices in the column from 3 to 8. Further work/discussion is needed later. The original ticket has it spanning 3 rows but the logic turned out more difficult than expected.

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-09-13 at 10 08 01 AM](https://github.com/linode/manager/assets/139489745/ef949418-e101-4605-bfe2-7f5aaa7e1768) | ![Screenshot 2023-09-13 at 4 45 09 PM](https://github.com/linode/manager/assets/139489745/0f06e74e-2b2d-4b3c-a435-28d5fa997d35) |

## How to test 🧪
1. Navigate to http://localhost:3000/firewalls
2. Ensure the column header is 'Devices' instead of 'Linodes'
3. Ensure additional nodes past the third row are not shown.
4. Ensure the links still bring you to the device landing pages
